src/uint/array.rs

```diff
@@ -35,6 +35,15 @@ macro_rules! impl_uint_array_encoding {
                     result
                 }
             }
+
+            #[cfg_attr(docsrs, doc(cfg(feature = "generic-array")))]
+            impl ArrayDecoding for GenericArray<u8, $bytes> {
+                type Output = super::$uint;
+
+                fn to_bigint(self) -> Self::Output {
+                    Self::Output::from_le_byte_array(self)
+                }
```

Comment:
This only supports little endian. Can you add two methods and cover both cases, e.g. `to_bigint_be` and `to_bigint_le` maybe?

Reply:
The trait is only supposed to be implemented for `GenericArray`, the conversion can be done either way, the result would be the same, I'm not sure what this would accomplish.

Reply:
In its current form the trait only supports a little endian conversion...

Reply:
Maybe I'm not understanding something here:
```rust
impl ArrayDecoding for GenericArray<u8, typenum::U32> {
    type Output = U384;

    fn into_bigint_le(self) -> Self::Output {
        Self::Output::from_le_byte_array(self)
    }

    fn into_bigint_be(self) -> Self::Output {
        // `GenericArray::as_be_slice` doesn't exist, I understand that's beside the point, but what should I do here?
        Self::Output::from_be_byte_array(self.as_be_slice())
    }
}
```
The outcome here would be exactly the same I assumed.

Reply:
The methods just describe how the data in the `GenericArray` is to be interpreted. No additional conversion is required.

```rust
impl ArrayDecoding for GenericArray<u8, typenum::U32> {
    type Output = U384;

    fn into_bigint_le(self) -> Self::Output {
        Self::Output::from_le_byte_array(self)
    }

    fn into_bigint_be(self) -> Self::Output {
        Self::Output::from_be_byte_array(self)
    }
}
```

Reply:
Ah ... right, I lost all context apparently :laughing:

Reply:
Done.